### ISSUE TITLE
add feature to deply packages without own definition

### ISSUE THREAD

**Issue by Flyingmana** [magento-hackathon/magento-composer-installer#93]

**Comment by razbakov** [comment]
is it for those which are not "type": "magento-module"?

**Comment by Flyingmana** [comment]
yes, I planned kind of a whiteliste for package names, so they the installer handles them ans they were magento-modules. I wonder if there would be needed much more at all, as we already have the `magento-map-overwrite`

**Comment by razbakov** [comment]
i mean all non-magento modules should be just moved to lib folder

**Comment by Flyingmana** [comment]
that would cause problem with people, who have a working composer setup.
Also nearly non php package would match the magento autoloading, causing possible odd sideeffects if accidentily a file path there matches the autoloading pattern.

**Comment by razbakov** [comment]
```
|- webroot
|  |- lib
|- vendor
```

for example i want to use https://github.com/symfony/Process.
the problem is that only content of webroot folder should be synced to production.

how would you solve it?

**Comment by Flyingmana** [comment]
I see, thats what I build the [libraryDeploy](https://github.com/Cotya/magento-composer-installer/blob/3.0.3/src/MagentoHackathon/Composer/Magento/Plugin.php#L277) for, it copys typical composer packages into a separate directory (so no conflict with autoloading) and creates a autoload map for it. Its not documented yet and also not finalized for working automagically.
And also its not intended for magento modules, which is why we have this Issue here :)
